These fail on 32-bit due to an overflow bug in the test program.

See test/regress_util.c:1448.

Upstream bug URL:

https://github.com/libevent/libevent/issues/452

diff --git a/test/regress_util.c b/test/regress_util.c
index ef6a1487..4de501fc 100644
--- a/test/regress_util.c
+++ b/test/regress_util.c
@@ -1413,9 +1413,9 @@ static struct date_rfc1123_case {
 	{  1323648000, "Mon, 12 Dec 2011 00:00:00 GMT"},
 #ifndef _WIN32
 	/** In win32 case we have max   "23:59:59 January 18, 2038, UTC" for time32 */
-	{  4294967296, "Sun, 07 Feb 2106 06:28:16 GMT"} /* 2^32 */,
+	//{  4294967296, "Sun, 07 Feb 2106 06:28:16 GMT"} /* 2^32 */,
 	/** In win32 case we have max "23:59:59, December 31, 3000, UTC" for time64 */
-	{253402300799, "Fri, 31 Dec 9999 23:59:59 GMT"} /* long long future no one can imagine */,
+	//{253402300799, "Fri, 31 Dec 9999 23:59:59 GMT"} /* long long future no one can imagine */,
 	{  1456704000, "Mon, 29 Feb 2016 00:00:00 GMT"} /* leap year */,
 #endif
 	{  1435708800, "Wed, 01 Jul 2015 00:00:00 GMT"} /* leap second */,
